src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -3367,7 +3284,7 @@ internal void HideUnhideNamespace(bool hide, string library, string namespc)
 
         internal void AddZeroTouchNodesToSearch(IEnumerable<FunctionGroup> functionGroups)
         {
-            var iDoc = InitializeIndexDocument();
+            var iDoc = LuceneSearchUtility.InitializeIndexDocumentForNodes();
```

Comment:
@RobertGlobant20 Not related to this task but maybe yours, can we explore way to append search info to existing index files? This would work either for new nodes or packages

Reply:
@QilongTang yes, right now I'm exploring how to append search info for the task of showing node-packages installed in the package search results.